eosdis-nasa/earthdata-pub-dashboard
# Edpub 1257 note visability

# Description

Dashboard has ability to limit note / comment visibility

## Spec


See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1257

## Linked PRs:
https://github.com/eosdis-nasa/earthdata-pub-api/pull/79/commits

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Start environment Locally
4. Create a new request & submit
5. As a DAAC Staff or DAAC Manager role verify ability to save a comment with limited visibility on the DAR page
6. Navigate to the conversations tab
7. Enter a conversation and verify ability to create a new comment with limited visibility

---

## Change Log

* Adds ability to limit note/comment visibility
